src/spec/create_extension_spec.py

```diff
@@ -14,36 +14,364 @@ def main():
+                doc="Link to the photodetector device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="dichroic_mirror",
+                doc="Link to the dichroic mirror device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="emission_filter",
+                doc="Link to the emission filter device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="excitation_filter",
+                doc="Link to the excitation filter device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+                quantity="?",
+            ),
+        ],
+    )
+
+    fiberphotometryresponse_series = NWBGroupSpec(
+        neurodata_type_def="FiberPhotometryResponseSeries",
+        neurodata_type_inc="TimeSeries",
+        doc="Extends TimeSeries to hold Fiber Photometry data.",
+        datasets=[
+            NWBDatasetSpec(
+                name="data",
+                doc="The data values. May be 1D or 2D. The first dimension must be time."
+                "The optional second dimension refers to the fiber that record the series.",
+                shape=(None, None),
+            ),
+            NWBDatasetSpec(
+                name="fiber_photometry_table_region",
+                doc="References row(s) of FiberPhotometryTable.",
+                neurodata_type_inc="DynamicTableRegion",
+                quantity="?",
+            ),
+        ],
+    )
+
+    commandedvoltage_series = NWBGroupSpec(
+        neurodata_type_def="CommandedVoltageSeries",
+        neurodata_type_inc="TimeSeries",
+        doc="Extends TimeSeries to hold a Commanded Voltage",
+        datasets=[
+            NWBDatasetSpec(
+                name="data",
+                doc="Voltages (length number timesteps) in unit volts.",
+                dtype="float",
+                shape=(None,),
+                attributes=[NWBAttributeSpec(name="unit", doc="data unit", value="volts", dtype="text")],
+            ),
+            NWBDatasetSpec(
+                name="frequency",
+                doc="Voltage frequency in unit hertz.",
+                dtype="float",
+                attributes=[NWBAttributeSpec(name="unit", doc="frequency unit", value="hertz", dtype="text")],
+                quantity="?",
+            ),
+        ],
+    )
+
+    multi_commanded_voltage = NWBGroupSpec(
```

Comment:
I don't think we need this anymore since commanded voltages are going directly in the FiberPhotometryTable.